addons/ekco/0.4.0/reboot/shutdown.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+export KUBECONFIG=/etc/kubernetes/kubelet.conf
+
+kubectl cordon $(hostname | tr '[:upper:]' '[:lower:]')
+
+# delete local pods with PVCs
+while read -r uid; do
+        pod=$(kubectl get pods --all-namespaces -ojsonpath='{ range .items[*]}{.metadata.name}{"\t"}{.metadata.uid}{"\t"}{.metadata.namespace}{"\n"}{end}' | grep $uid )
+        kubectl delete pod $(echo $pod | awk '{ print $1 }') --namespace=$(echo $pod | awk '{ print $3 }') --wait=false
+done < <(lsblk | grep '^rbd[0-9]' | awk '{ print $7 }' | awk -F '/' '{ print $6 }')
+
+# delete local pods using the Ceph filesystem
+while read -r uid; do
+        pod=$(kubectl get pods --all-namespaces -ojsonpath='{ range .items[*]}{.metadata.name}{"\t"}{.metadata.uid}{"\t"}{.metadata.namespace}{"\n"}{end}' | grep $uid )
+        kubectl delete pod $(echo $pod | awk '{ print $1 }') --namespace=$(echo $pod | awk '{ print $3 }') --wait=false
+done < <(cat /proc/mounts | grep ':6789:/' | grep -v globalmount | awk '{ print $2 }' | awk -F '/' '{ print $6 }')
```

Comment:
The `| grep -v globalmount` is the only thing new in here. Any node that has a pod mounting the shared fs now has an extra mount:

```
areed@areed-aka-21rl:~/kurl$ cat /proc/mounts | grep 6789
10.96.0.199:6789,10.96.2.45:6789,10.96.1.230:6789:/volumes/csi/csi-vol-cf576638-f93e-11ea-a492-8626cbbad91b/a5c1309b-8773-4d56-906f-8c4deda3f8ea /var/lib/kubelet/plugins/kubernetes.io/csi/pv/pvc-a4829b95-6bec-4d58-a7fa-4b76378eb676/globalmount ceph rw,relatime,name=csi-cephfs-node,secret=<hidden>,acl,mds_namespace=rook-shared-fs 0 0
10.96.0.199:6789,10.96.2.45:6789,10.96.1.230:6789:/volumes/csi/csi-vol-cf576638-f93e-11ea-a492-8626cbbad91b/a5c1309b-8773-4d56-906f-8c4deda3f8ea /var/lib/kubelet/pods/6341d404-91b8-4268-893f-aec1cf2c114d/volumes/kubernetes.io~csi/pvc-a4829b95-6bec-4d58-a7fa-4b76378eb676/mount ceph rw,relatime,name=csi-cephfs-node,secret=<hidden>,acl,mds_namespace=rook-shared-fs 0 0
```